<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>R3R-126 SWL Logger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="callsignEdit_1">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>CALLSIGN</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="hamQTHBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>HamQTH</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QTableWidget" name="table">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Freq</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mode</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Call1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Call2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RST1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RST2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Comment1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Comment2</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="QTHEdit_1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>QTH</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="locatorEdit_1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>LOCATOR</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="realTimeCheckBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Real Time</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTimeEdit" name="startTimeEdit">
      <property name="displayFormat">
       <string>HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QLineEdit" name="QTHEdit_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>QTH</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLineEdit" name="nameEdit_1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>NAME</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QLineEdit" name="callsignEdit_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>CALLSIGN</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLineEdit" name="RSTEdit_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="placeholderText">
       <string>RST</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="clearButton">
      <property name="text">
       <string>Стереть</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="modeBox">
      <item>
       <property name="text">
        <string>SSB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RTTY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DIGI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SSTV</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="3" colspan="3">
     <widget class="QLineEdit" name="commentEdit_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>COMMENT</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="RSTEdit_1">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="placeholderText">
       <string>RST</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLineEdit" name="beeEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QSpinBox" name="freqBox">
      <property name="maximum">
       <number>500000</number>
      </property>
      <property name="value">
       <number>7000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="7" column="3" colspan="3">
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLineEdit" name="commentEdit_1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>COMMENT</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLineEdit" name="locatorEdit_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>LOCATOR</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Между:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="3">
     <widget class="QPushButton" name="nonClearButton_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Не стирать</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="3" colspan="3">
     <widget class="QLineEdit" name="nameEdit_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>NAME</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QPushButton" name="nonClearButton_1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Не стирать</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTimeEdit" name="endTimeEdit">
      <property name="displayFormat">
       <string>HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="2">
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionADIF"/>
    <addaction name="action_eQSL"/>
    <addaction name="separator"/>
    <addaction name="actionClearBase"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionChangeBand"/>
    <addaction name="actionChangeMode"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionADIF">
   <property name="text">
    <string>Экспорт в ADIF</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="action_from_CSV">
   <property name="text">
    <string>Открыть CSV</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_to_CSV">
   <property name="text">
    <string>Сохранить CSV</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionChangeBand">
   <property name="text">
    <string>Сменить диапозон</string>
   </property>
   <property name="shortcut">
    <string>Alt+B</string>
   </property>
  </action>
  <action name="actionChangeMode">
   <property name="text">
    <string>Сменить режим</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionClearBase">
   <property name="text">
    <string>Очистить историю</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Информация</string>
   </property>
  </action>
  <action name="action_eQSL">
   <property name="text">
    <string>Экспорт на eQSL</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
